src/plugins/vis_types/pie/public/components/visualization_noresults.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiEmptyPrompt, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export const VisualizationNoResults = ({ hasNegativeValues = false }) => {
+  return (
+    <EuiEmptyPrompt
+      iconType="visualizeApp"
+      iconColor="default"
+      data-test-subj="pieVisualizationError"
+      body={
+        <EuiText size="xs">
+          {hasNegativeValues
+            ? i18n.translate('visTypePie.negativeValuesFound', {
+                defaultMessage:
+                  "Partition charts can't render with negative values. Try a different visualization type.",
```

Comment:
Do we use `Partitions` anywhere else? I think using explicitly Pie or donuts it's clearer

```suggestion
                  "Pie/donut charts can't render with negative values. Try a different visualization type.",
```

Reply:
Robert suggested this but I prefer your suggestion. I will change this